src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,95 @@
+import logo from "./logo.svg";
+
+import {
+  Container,
+  Grid,
+  Box,
+  Button,
+  Typography,
+  Link,
+  Avatar,
+} from "@mui/material";
+import { Facebook, GitHub, LinkedIn } from "@mui/icons-material";
+
+function Copyright() {
+  return (
+    <Typography variant="body2" color="text.secondary">
+      {"Copyright © "}
```

Comment:
why do you need curly braces?

Reply:
its for whitespace, removed curly braces and added {" "} instead